packages/snage/src/config/load.ts

```diff
@@ -128,6 +134,25 @@ const createSingleLinkProvider = (fields: Field[]) => (index: number, link: Link
     );
 };
 
+const createStyleProvider = (fields: Field[], styles: ConditionalStyle[]): E.Either<string, CSSProvider> => {
+    const parser = createParser(fields);
+
+    return pipe(
+        A.array.traverse(E.either)(styles, ({on, css}) =>
+            E.either.bimap(
+                parser(on),
+                (e) => `Invalid expression "${on}" ${JSON.stringify(e)}`,
+                (expression) => ({matcher: createMatcher(expression, fields), css} as const)
```

Comment:
Dammit, we've forgotten to check out the `Optional`-variant for `CSSProvider`. Now we'll never know if it is better.